examples/jsm/loaders/SDFLoader.js

```diff
@@ -0,0 +1,195 @@
+import {
+	Loader,
+	FileLoader,
+	Group,
+	Vector3,
+	Color,
+	SphereGeometry,
+	CylinderGeometry,
+	MeshLambertMaterial,
+	InstancedMesh,
+	Matrix4,
+	Quaternion
+} from 'three';
+
+const DEFAULT_VDW_RADIUS = {
+	H: 0.31, C: 0.76, N: 0.71, O: 0.66, F: 0.57,
+	P: 1.07, S: 1.05, Cl: 1.02, Br: 1.2, I: 1.39
+};
+
+const DEFAULT_ELEMENT_COLOR = {
+	H: 0xffffff, C: 0x909090, N: 0x3050f8, O: 0xff0d0d, F: 0x90e050,
+	P: 0xff8000, S: 0xffff30, Cl: 0x1ff01f, Br: 0xa62929, I: 0x940094
+};
+
+/**
+	* SDFLoader — A loader for MDL Molfile / SDF chemical structure files.
+	*
+	* The loader parses the textual SDF data and returns a `THREE.Group` containing two
+	* `THREE.InstancedMesh` children: one for the atoms (spheres) and one for the bonds (cylinders).
+	*
+	* In addition to efficient rendering via instancing, the loader allows customisation of element
+	* colours and van-der-Waals radii via the `setElementData` method.
+	*
+	* Example usage:
+	* ```js
+	* import { SDFLoader } from 'three/addons/loaders/SDFLoader.js';
+	*
+	* const loader = new SDFLoader();
+	* loader.load( 'models/sdf/benzene.sdf', group => {
+	*     scene.add( group );
+	* } );
+	* ```
+	*
+	* @see examples/webgl_loader_sdf
+	*/
+class SDFLoader extends Loader {
```

Comment:
I vote to remove `SDFLoader` and implement the JSON parsing right in the example. The loader contains too much application specific code.

Reply:
The SDF parsing library could be imported like in other examples:

https://github.com/mrdoob/three.js/blob/c6e73de1367ddd47fa02e548a9fcc06153bbefa7/examples/webgl_loader_ifc.html#L26-L28